File: Assignment 5/Problem1.java
// Accept number from user and display multiplication of that factor

import java.lang.*;
import java.util.*;

class Multiplication
{
	public int iValue;
	
	public Multiplication(int iNo)
	{
		this.iValue =iNo;		
	}
	
	public int MultiFactor()
	{
		int iCnt = 0;
		int iSum = 1;
		
		for(iCnt=1;iCnt<iValue;iCnt++)
		{
			if(iValue%iCnt == 0)
			{
				iSum =iSum*iCnt;
			}
		}
			return iSum;
	}
}

class Assignment1
{
	public static void main(String arr[])
	{
		int iValue=0;
		
		Scanner sobj = new Scanner (System.in);
		
		System.out.println("Enter Number");
		iValue = sobj.nextInt();
		
		Multiplication mobj = new Multiplication(iValue);
		
		int iRet = mobj.MultiFactor();
		
		System.out.println("Multiplication of Fator is: "+iRet);
}
}

File: Assignment4/Assignment4/problem1.c
//Accept numbers from user and display multiplication of its factor.
//Number -12
// Output - 144(1*2*3*4*6)


#include<stdio.h>

int MultiFactor(int iNo)
{
  int iCnt =0;
  int iMulti =1;
  
  if(iNo<0)
  {
  	iNo=-iNo;
	 	
  }
 	
  
  for(iCnt=1;iCnt<=iNo;iCnt++)
  {   
	if(((iNo%iCnt)==0) && (iNo!=iCnt) )
  	{   
  	    iMulti=iMulti*iCnt;
  	}
  }   

return iMulti;
}

int main()
{
  int iValue =0;
  int iRet =0;
 
  printf("Enter Number\n");
  scanf("%d",&iValue);
   
   iRet =MultiFactor(iValue);
   
   printf("%d\n",iRet);

  return 0;
}

